yarn-project/stdlib/src/interfaces/aztec-node.ts

```diff
@@ -577,6 +580,11 @@ export const AztecNodeApiSchema: ApiSchemaFor<AztecNode> = {
 
   getValidatorsStats: z.function().returns(ValidatorsStatsSchema),
 
+  getValidatorStats: z
+    .function()
+    .args(z.string(), optional(schemas.BigInt), optional(schemas.BigInt))
```

Comment:
we should have an EthAddress schema somewhere, i imagine this will panic later if this string is not well formed